dig/chain.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "dig",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "Dig Chain",
+  "chain_id": "dig-1",
+  "bech32_prefix": "dig",
+  "daemon_name": "digd",
+  "node_home": "$HOME/.digd",
+  "genesis": {
+    "genesis_url":  "https://raw.githubusercontent.com/notional-labs/dig/master/networks/mainnets/dig-1/genesis.json"
+  },
+  "key_algos": [
+    "secp256k1",
+    "eth-secp256k1"
```

Comment:
This is supported?

Reply:
Hi @sunnya97, eth-escp256k1 is supported. We are working to support clients using eth addresses